src/vs/workbench/contrib/files/browser/views/explorerViewer.ts

```diff
@@ -1321,36 +1321,43 @@ export class FileDragAndDrop implements ITreeDragAndDrop<ExplorerItem> {
 		return this.workspaceEditingService.updateFolders(0, workspaceCreationData.length, workspaceCreationData);
 	}
 
-	private async doHandleExplorerDrop(source: ExplorerItem, target: ExplorerItem, isCopy: boolean): Promise<void> {
+	private async doHandleExplorerDrop(sources: ExplorerItem[], target: ExplorerItem, isCopy: boolean): Promise<void> {
 		// Reuse duplicate action if user copies
+		const files = [];
```

Comment:
Please type this array

Reply:
Also can we use a more explicit name, `files` sounds too general

Reply:
The array is typed and used a more explicit name, I think.